src/jesse_schema_validator.hrl

```diff
@@ -122,6 +122,7 @@
 -define(any_schemas_not_valid,       'any_schemas_not_valid').
 -define(not_multiple_of,             'not_multiple_of').
 -define(not_one_schema_valid,        'not_one_schema_valid').
+-define(more_than_one_schema_valid,  'more_than_one_schema_valid').
```

Comment:
I thought adding this more specific error would be nice, but I can remove it if you think it's not worth it.